src/main/kotlin/sliv/tool/catalogue/view/CatalogueView.kt

```diff
@@ -1,10 +1,171 @@
 package sliv.tool.catalogue.view
 
 import javafx.collections.FXCollections
+import javafx.geometry.Insets
+import javafx.geometry.Pos
+import javafx.scene.control.CheckBox
+import javafx.scene.control.RadioButton
+import javafx.scene.control.SelectionMode
+import javafx.scene.control.Toggle
+import javafx.scene.control.ToggleGroup
+import javafx.scene.layout.Priority
+import sliv.tool.catalogue.*
+import sliv.tool.catalogue.controller.CatalogueController
+import sliv.tool.catalogue.model.CatalogueField
+import sliv.tool.catalogue.model.ViewFormat
 import tornadofx.*
 
 class CatalogueView : View() {
-    override val root = hbox {
-        listview(FXCollections.observableArrayList("1", "2", "3"))
+    enum class SelectionState {
+        ALL,
+        NONE,
+        PART
     }
-}
\ No newline at end of file
+
+    companion object {
+        private val initialViewFormat = ViewFormat.FILE_NAME
```

Comment:
Uppercase underscore-separated name should be used

Reply:
As I remember in the standard kotlin style guide it is ok
From https://kotlinlang.org/docs/coding-conventions.html#property-names:
"_For enum constants, it's OK to use either uppercase underscore-separated names (screaming snake case) (enum class Color { RED, GREEN }) or upper camel case names, depending on the usage_"

Reply:
We use `upper camel case name` everywhere))